### DISCUSSION TITLE
Styling Lyrics on Playback

### DISCUSSION THREAD

**Discussion by Linguiniotta** [jcorporation/myMPD#1378] [Q&A] [answered]
Thanks for this wonderful project. I've recently found it and I've been loving it!

Is there a way to stylize the scrolling lyrics? I was looking for options to emphasize (bolder text) the current lyrics, and increase the overall font size of the displayed lyrics.

**Comment by jcorporation** [comment, marked as answer]
Currently there is no setting for this, but you can edit the CSS and compile myMPD yourself.

I see no big demand to add settings for this special use case. What I can imagine is the option to define custom css without the requirement to recompile myMPD. Would this fulfill your requirements.

**Reply by Linguiniotta** [reply]
I settled in using custom CSS with this Firefox [extension](https://addons.mozilla.org/en-US/firefox/addon/styl-us)
